docs/architecture/baking.md

```diff
@@ -31,18 +31,29 @@ To become a baker, you must create an account to act as a _delegate_, which is a
 Delegates temporarily stake tez as a security deposit to ensure that they are acting according to the rules of the protocol.
 This stake stays locked for a certain number of cycles before the delegate can begin baking.
 Similarly, when bakers stop baking, their stake is unlocked after a certain number of cycles.
+While it is locked, the security deposit may be slashed by the protocol if the baker misbehaves (e.g., proposes or attests two different blocks for the same level).
 
-The delegate must stake at least least 6,000 tez, either from its own account or from tez that delegators delegate to it.
-The more tez a delegate has, both from its own account and from delegators, the more likely it is to be selected to bake a block and thus receive the rewards.
+The delegate must stake at least least 600 tez from its own account,
+and must also represent at least 6,000 tez of "active stake", which is computed
```

Comment:
It seems you are reviewing an old version of the page. This paragraph has been completely rewritten in commit f030737899e7feb4ccf76bb77717fee167790737 so that it doesn't mention anymore the 600 tez, nor the active stake. Please update your branch and review again.